### DISCUSSION TITLE
Could not dynamically load CUDA

### DISCUSSION THREAD

**Discussion by 641i130** [ppy/osu#17566] [Q&A]
### Type

Crash to desktop

### Bug description

When playing the game, randomly when I select a beatmap to play, it freezes on loading then crashes with this error report:
I'm unsure if it's related to the beatmaps because it seems to happen very randomly when choosing maps to play.
```
ALSA lib control.c:1528:(snd_ctl_open_noupdate) Invalid CTL jack
Failed to open VDPAU backend libvdpau_radeonsi.so: cannot open shared object file: No such file or directory
[AVHWDeviceContext @ 0x7fb79875f680] VDPAU device creation on X11 display :0 failed.
[AVHWDeviceContext @ 0x7fb79800d480] Cannot load libcuda.so.1
[AVHWDeviceContext @ 0x7fb79800d480] Could not dynamically load CUDA
[AVHWDeviceContext @ 0x7fb79877d500] Cannot load libcuda.so.1
[AVHWDeviceContext @ 0x7fb79877d500] Could not dynamically load CUDA
Failed to open VDPAU backend libvdpau_radeonsi.so: cannot open shared object file: No such file or directory
[AVHWDeviceContext @ 0x7fb79c05b140] VDPAU device creation on X11 display :0 failed.
[AVHWDeviceContext @ 0x7fb79c01c380] Cannot load libcuda.so.1
[AVHWDeviceContext @ 0x7fb79c01c380] Could not dynamically load CUDA
[AVHWDeviceContext @ 0x7fb79c01c380] Cannot load libcuda.so.1
[AVHWDeviceContext @ 0x7fb79c01c380] Could not dynamically load CUDA
Failed to open VDPAU backend libvdpau_radeonsi.so: cannot open shared object file: No such file or directory
[AVHWDeviceContext @ 0x7fb7a064c140] VDPAU device creation on X11 display :0 failed.
[AVHWDeviceContext @ 0x7fb7a0068b00] Cannot load libcuda.so.1
[AVHWDeviceContext @ 0x7fb7a0068b00] Could not dynamically load CUDA
corrupted size vs. prev_size
```
This could be related to a driver issue when trying to load CUDA on an AMD GPU system (I'm using an AMD GPU). I don't have anything CUDA related installed either. 

### Version

2022.327.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/8384665/database.log)
[network.log](https://github.com/ppy/osu/files/8384667/network.log)
[performance.log](https://github.com/ppy/osu/files/8384668/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8384669/runtime.log)

**Comment by 641i130** [comment]
I did some research on that error and it could be related to me not having `libva-vdpau-driver` installed... I'm rebooting to test this. (on arch based distro)

**Reply by peppy** [reply]
Whenever you get an error on linux that looks like it is in a driver, please consider researching and looking or a solution before posting against an application (like osu!). It's not an issue that we generally can support, and up to you as a linux user to resolve yourself.
